LAKMobile/src/api/data.ts

```diff
@@ -1,13 +1,21 @@
+export type JobStatus = 'CREATED' | 'ASSIGNED' | 'COMPLETED';
 export interface JobData {
+    _id: string,
     title: string;
     clientName: string;
     phoneNumber?: string;
     deliveryDate: string;
     pickupLocation: string;
     dropoffLocation: string;
     packageQuantity?: number;
-    applicants?: number;
-    status: 'CREATED' | 'ASSIGNED' | 'COMPLETED' ;
+    status: JobStatus;
+    imageIds: string[];
+}
+
+export interface JobOwnerView extends JobData{
```

Comment:
Created JobOwnerView and JobApplicantView because depending on the mode, there can be different arguments